### PR TITLE
Paint blending

### PR DESCRIPTION
**Pull request by IkeRijsdijk**

### CODE REVIEW

File: core/src/main/java/nl/tudelft/contextproject/core/screens/GameScreen.java [hunk 1]
```diff
@@ -12,6 +12,7 @@
 import com.badlogic.gdx.math.Vector2;
 import nl.tudelft.contextproject.core.Main;
 import nl.tudelft.contextproject.core.config.Constants;
+import nl.tudelft.contextproject.core.entities.Colour;
```
Comment (Erackron) [inline core/src/main/java/nl/tudelft/contextproject/core/screens/GameScreen.java]: Try not to leave unused import statements. PMD and IntelliJ will tell you if an import statement is unused.

File: core/src/main/java/nl/tudelft/contextproject/core/entities/Colour.java [hunk 2]
```diff
@@ -1,21 +1,103 @@
...
-    SALMON(255, 181, 153), BLACK(0, 0, 0);
+    RED(1, 0, 0, -16776961),
+    YELLOW(1, 1, 0, -65281),
+    BLUE(0, 0, 1, 65535),
+    GREEN(34 / 255f, 170 / 255f, 34 / 255f, 581575423),
+    PURPLE(128 / 255f, 0, 128 / 255f, -2147450625),
+    ORANGE(255 / 255f, 165 / 255f, 0, -5963521),
+    BLACK(0, 0, 0, 255);
+
+    private Color libgdxColor;
```
Comment (LiamClark) [inline core/src/main/java/nl/tudelft/contextproject/core/entities/Colour.java]: these field could be declared final, mutability in enum classes is generally best if avoided.

File: core/src/main/java/nl/tudelft/contextproject/core/entities/Colour.java [hunk 3]
```diff
@@ -1,21 +1,103 @@
...
+        return BLACK;
+    }
+
+    /**
+     * Method that checks if two colours are complementary.
+     * @param colours Collection that contains the two colours that need to be compared.
+     * @return Boolean value.
+     */
+    public static boolean areComplementary(Collection<Colour> colours) {
+        if (colours.size() != 2) {
```
Comment (MitchellHop) [inline core/src/main/java/nl/tudelft/contextproject/core/entities/Colour.java:54]: is it not as effective to put colours.size() == 2 in the if statement that returns true? it also shortens the method a little

Comment (martinck) [inline core/src/main/java/nl/tudelft/contextproject/core/entities/Colour.java:54]: These checks (another one in the combine method), could be changed to throwing exceptions upon incorrect colours.size

**Pull request merged**